templates/level-struktur/level-struktur.css

```diff
@@ -0,0 +1,9 @@
+body.level-struktur {
+    background-color: black;
+    color: white;
+  }
+
+body.level-struktur h1 > strong{
+ font-size: var(--heading-font-size-xxl);
+ font-weight: var(--font-weight-bold);
+}
```

Comment:
This should be inheriting from the styles.css already except for the color
![image](https://github.com/aemsites/888de/assets/3883795/7882a2f7-b1a4-4045-b01c-e23747ccc51e)